doc/irb/indexes.rdoc

```diff
@@ -0,0 +1,193 @@
+  return values.
+- <tt>--noecho-on-assignment</tt>: Don't print ({echo}[rdoc-ref:IRB@Return-Value+Printing+-28Echoing-29])
+  result on assignment.
+- <tt>--noinspect</tt>: Don't se method +inspect+ for printing ({echoing}[rdoc-ref:IRB@Return-Value+Printing+-28Echoing-29])
+  return values.
+- <tt>--nomultiline</tt>: Don't use the multiline editor as the {input method}[rdoc-ref:IRB@Input+Method].
+- <tt>--noprompt</tt>: Don't print {prompts}[rdoc-ref:IRB@Prompt+and+Return+Formats].
+- <tt>--noscript</tt>:  Treat the first command-line argument as a normal
+  {command-line argument}[rdoc-ref:IRB@Initialization+Script],
+  and include it in +ARGV+.
+- <tt>--nosingleline</tt>: Don't use the singleline editor as the {input method}[rdoc-ref:IRB@Input+Method].
+- <tt>--noverbose</tt>Don't print {verbose}[rdoc-ref:IRB@Verbosity] details.
+- <tt>--prompt _mode_</tt>, <tt>--prompt-mode _mode_</tt>:
+  Set {prompt and return formats}[rdoc-ref:IRB@Prompt+and+Return+Formats];
+  +mode+ may be a {pre-defined prompt}[rdoc-ref:IRB@Pre-Defined+Prompts]
+  or the name of a {custom prompt}[rdoc-ref:IRB@Custom+Prompts].
+- <tt>--script</tt>: Treat the first command-line argument as the path to an
+  {initialization script}[rdoc-ref:IRB@Initialization+Script],
+  and omit it from +ARGV+.
+- <tt>--simple-prompt</tt>, <tt>--sample-book-mode</tt>:
+  Set prompt mode to {:SIMPLE}[rdoc-ref:IRB@Pre-Defined+Prompts].
+- <tt>--singleline</tt>: Use the singleline editor as the {input method}[rdoc-ref:IRB@Input+Method].
+- <tt>--tracer</tt>: Use {Tracer}[rdoc-ref:IRB@Tracer] to print a stack trace for each input command.
+- <tt>--truncate-echo-on-assignment</tt>: Print ({echo}[rdoc-ref:IRB@Return-Value+Printing+-28Echoing-29])
+  truncated result on assignment.
+- <tt>--verbose</tt>Print {verbose}[rdoc-ref:IRB@Verbosity] details.
+- <tt>-v</tt>, <tt>--version</tt>: Print the {IRB version}[rdoc-ref:IRB@Version].
+- <tt>-h</tt>, <tt>--help</tt>: Print the {IRB help text}[rdoc-ref:IRB@Help].
+- <tt>--</tt>: Separate options from {arguments}[rdoc-ref:IRB@Command-Line+Arguments]
+  on the command-line.
+
+=== Index of \IRB.conf Entries
+
+These are the keys for hash \IRB.conf entries, with links to explanatory text;
+for each entry that is pre-defined, the initial value is given:
+
+- <tt>:AP_NAME</tt>: \IRB {application name}[rdoc-ref:IRB@Application+Name];
+  initial value: <tt>'irb'</tt>.
+- <tt>:AT_EXIT</tt>: Array of hooks to call
+  {at exit}[rdoc-ref:IRB@IRB];
+  initial value: <tt>[]</tt>.
+- <tt>:AUTO_INDENT</tt>: Whether {automatic indentation}[rdoc-ref:IRB@Automatic+Indentation]
+  is enabled; initial value: +true+.
+- <tt>:BACK_TRACE_LIMIT</tt>: Sets the {back trace limit}[rdoc-ref:IRB@Tracer];
+  initial value: +16+.
+- <tt>:COMMAND_ALIASES</tt>: Defines input {command aliases}[rdoc-ref:IRB@Command+Aliases];
+  initial value:
+
+    {
+      "$": :show_source,
+      "@": :whereami,
+      break: :irb_break,
+      catch: :irb_catch,
+      next: :irb_next
```

Comment:
I recently hide these from users so we can remove them.

Reply:
@BurdetteLamar These should be removed

Reply:
Removed.